src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -294,48 +299,15 @@ public async Task<ActionResult> HideWizard(string hashedAccountId)
         [ChildActionOnly]
         public ActionResult Row1Panel1(AccountDashboardViewModel model)
         {
-            var viewModel = new PanelViewModel<AccountDashboardViewModel> { ViewName = "CheckFunding", Data = model };
-            if (model.AgreementsToSign)
-            {
-                viewModel.ViewName = "SignAgreement";
-            }
-            else if (model.RecentlyAddedReservationId != null
-                || model.AccountViewModel?.AccountLegalEntities?.FirstOrDefault()?.ReservedFundings?.Any() == true)
-            {
-                viewModel.ViewName = "FundingComplete";
-
-                //todo: no need to return everything in the event in AccountDto, just what we need to display (probably only save what we need to show also)
-                //todo: accountDto now mixed concrete/interfaces, which is inconsistent
-
-                if (model.RecentlyAddedReservationId != null)
-                {
-                    var legalEntity = model.AccountViewModel?.AccountLegalEntities
-                        ?.FirstOrDefault(ale => ale.ReservedFundings?.Any(rf => rf.ReservationId == model.RecentlyAddedReservationId) == true);
-
-                    model.ReservedFundingToShowLegalEntityName = legalEntity?.LegalEntityName;
-
-                    // would be better to create new model to contain what the panel needs to show,
-                    // but we'll be replacing this with displaying all reserved funds anyway
-                    model.ReservedFundingToShow =
-                        legalEntity?.ReservedFundings?.FirstOrDefault(rf =>
-                            rf.ReservationId == model.RecentlyAddedReservationId);
-                }
-
-                if (model.ReservedFundingToShow == null)
-                {
-                    var legalEntity = model.AccountViewModel?.AccountLegalEntities?.First();
-                    model.ReservedFundingToShowLegalEntityName = legalEntity?.LegalEntityName;
-                    model.ReservedFundingToShow = legalEntity?.ReservedFundings?.First();
-                }
-            }
+            var viewModel = _homepagePanelViewHelper.GetNextAction(model);
             return PartialView(viewModel);
-        }       
+        }
 
         [ChildActionOnly]
         public ActionResult Row1Panel2(AccountDashboardViewModel model)
         {
             var viewModel = new PanelViewModel<AccountDashboardViewModel> { ViewName = "ProviderPermissions", Data = model };
-            if (model.AgreementsToSign)
+            if (model.PayeSchemeCount == 0 || model.AgreementsToSign)
```

Comment:
This should be a single flag/property on the model so the code here is something like

if(model.CanAddProviderPermissions)
{
}